docs/maintainer/infrastructure.md

```diff
@@ -388,3 +418,226 @@ If this is not done, then the output validation process will block the package f
+- [`help-ruby`](https://github.com/orgs/conda-forge/teams/help-ruby)
+- [`miniforge`](https://github.com/orgs/conda-forge/teams/miniforge)
+- [`all-members`](https://github.com/orgs/conda-forge/teams/all-members)
+
+#### Configuration
+
+- [`conda-forge/.github`](https://github.com/conda-forge/.github): Organization-wide configuration, profile information, etc.
+
+#### Bot accounts
+
+- [`conda-forge-admin`](https://github.com/conda-forge-admin)
+- [`conda-forge-bot`](https://github.com/conda-forge-bot)
+- [`conda-forge-coordinator`](https://github.com/conda-forge-coordinator)
+- [`conda-forge-daemon`](https://github.com/conda-forge-daemon)
+- [`conda-forge-linter`](https://github.com/conda-forge-linter)
+- [`conda-forge-manager`](https://github.com/conda-forge-manager)
+- [`conda-forge-status`](https://github.com/conda-forge-status)
+- [`regro-cf-autotick-bot`](https://github.com/regro-cf-autotick-bot)
+
+:::info
+These accounts exist but are not in active usage anymore:
+
+- [`conda-forge-drone-ci`](https://github.com/conda-forge-drone-ci)
+
+:::
+
+#### Apps
+
+- `conda-forge-curator`
+- `conda-forge-webservices`
+
+:::info
+These apps exist but are not in active usage anymore:
+
+- `conda-forge drone instance`
+
+:::
+
+#### Workflows
+
+- [`beckermr/turnstyle-python`](https://github.com/beckermr/turnstyle-python): Prevents multiple CI jobs from running in parallel to avoid race conditions.
+- [`conda-forge/automerge-action`](https://github.com/conda-forge/automerge-action)
+- [`conda-forge/github-app-token`](https://github.com/conda-forge/github-app-token) (fork)
+- [`conda-forge/webservices-dispatch-action`](https://github.com/conda-forge/webservices-dispatch-action)
+
+### Continuous integration
+
+:::tip See also
+Refer to the [`conda-forge.yml` documentation](/docs/maintainer/conda_forge_yml/) to learn how to configure your CI providers.
+:::
+
+#### Azure Pipelines
+
+- 🌐 https://dev.azure.com/conda-forge/feedstock-builds/_build
+- 📍 Available on all feedstocks
+- 🛠 Provides [Microsoft-hosted runners](https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml) (x64 Linux, macOS and Windows)
+- 🔒 Needs access to Anaconda.org (cf-staging)
+
+conda-forge benefits from the generously offered Microsoft-hosted runners.
+
+#### Travis CI
+
+- 🌐 https://app.travis-ci.com/github/conda-forge
+- 📍 Available on all feedstocks
+- 🛠 Provides [native Linux aarch64 and ppc64le runners](https://docs.travis-ci.com/user/reference/overview/)
+- 🔒 Needs access to Anaconda.org (cf-staging)
+
+#### Cirun
+
+- 🌐 https://cirun.io
+- 📍 Available on selected feedstocks only
+- 🛠 Provides several architectures (depending on feedstock configuration)
+- 🔒 Needs access to Anaconda.org (cf-staging) and the configured backend
+
+Configured with `@conda-forge-daemon`.
+
+Organization-wide configuration can be found in the [`.cirun` repository](https://github.com/conda-forge/.cirun).
```

Comment:
An admonition here about the GPU Server pointing to `Quansight/open-gpu-server` would be nice too.